### ISSUE TITLE
关于拉取instagram图片的node程序能否开源？

### ISSUE THREAD

**Issue by mrdream24** [litten/hexo-theme-yilia#103]
看博客文章说在hexo g的时候跑一个node程序将instagram图片拉到本地保存，能否开源这个node程序，让大家学习一下，如不方便，能否写篇博客简单介绍一下原理呢？知道是不情之请，所以如果可以非常感谢，如果不行，那也情理之中。

**Comment by korvin101** [comment]
这个？http://litten.github.io/2014/03/03/instagram-api-ex/

**Comment by mrdream24** [comment]
是的。“刚实现这个功能的第三天，ins就被天朝封了，所以你看到没法显示是正常的，也没有太好的办法。我的相册可以显示，是因为我单独跑了个nodejs任务，每次执行hexo g命令时，通过一个代理把ins的照片拉到本地保存，再执行hexo d推送上去”。这个nodejs任务。

**Comment by korvin101** [comment]
@mrdream24 拉取照片是在执行hexo g任务时吗？如果是，那我拉取时电脑科学上网能不能拉到
